.flake8

```diff
@@ -15,18 +15,20 @@ ignore =
     W504,      # line break before binary operator (again, I guess).
     S101,      # Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.
 
-# F401:  unused import.
-# F403:  cannot detect unused vars if we use starred import
+# F401: unused import.
+# F403: cannot detect unused vars if we use starred import
 # FS003: f-string missing prefix.
 # R102: unnecessary parenthesis on raised exception (raises false positives in places)
+# S106: posible hardcoded password (we dont care about this in the tests)
+# D104, D103, D101, D100: Missing docstring in public function/class/module (ignore in tests)
```

Comment:
We should add docstrings.

Reply:
They are too much hassle for what they bring. The test name should describe extacly what the test tests. If any further info is needed, there are comments explaining it

Reply:
Hmm. It would enable some of the tests to be clearer and have shorter names IMO.

Reply:
It's going to be more of a PITA cause I would have to write a docstring for each of the 1000 tests...

Reply:
:-)

Reply:
We should still consider doing this at some point.